pkg/models/dashboard_version.go

```diff
@@ -48,6 +48,7 @@ type DashboardVersionMeta struct {
 type DashboardVersionDTO struct {
 	Id            int64     `json:"id"`
 	DashboardId   int64     `json:"dashboardId"`
+	UID           string    `json:"uid"`
```

Comment:
I would suggest better rename it to `DashboardUID` to be clearer.
```suggestion
	DashboardUID           string    `json:"dashboardUid"`
```